aiida/repository/repository.py

```diff
@@ -417,6 +417,14 @@ def delete_object(self, path: FilePath, hard_delete: bool = False):
         directory = self.get_directory(path.parent)
         directory.objects.pop(path.name)
 
+    def delete(self):
+        """Delete the entire repository.
+
+        .. important:: this will not just delete the contents of the repository but also the repository itself. For
```

Comment:
```suggestion
        .. important:: This will not just empty the contents of the repository but remove any trace of the repository from the file system. For
```

Reply:
Cannot say file system because this `Repository` class can service any repo, also an object store for example.